lib/scenic/view.rb

```diff
@@ -44,10 +44,10 @@ def ==(other)
     def to_schema
       materialized_option = materialized ? "materialized: true, " : ""
       <<-DEFINITION
-
   create_view :#{name}, #{materialized_option} sql_definition: <<-\SQL
     #{definition.indent(2)}
   SQL
+
       DEFINITION
```

Comment:
I've played with stip_heredoc so much and this is the most reasonable looking output I've been able to get for schema.rb.